wbb/modules/rice.py

```diff
@@ -1,27 +1,26 @@
 from pyrogram import filters
 from pyrogram.types import Message
 from wbb import app
-from wbb.utils import cust_filter
-
 
 __MODULE__ = "Rice"
 __HELP__ = "/rice - To Forward Your Linux Rice To DE_WM's" \
-           "Rice Gallery only works in [DE_WM](t.me/de_wm)"
+           "Rice Gallery only works in [DE_WM](t.me/DE_WM)"
 
```

Comment:
what about this? @thehamkercat 
those commands does not exist anymore, it should not be appeared in `/help`

Reply:
solved